### ISSUE TITLE
Crash when editing file name

### ISSUE THREAD

**Issue by yangl1996** [mssun/passforios#94]
```
Device: iPhone 6s
OS: iOS 10.3.2 (14F89)
App Build: 0.2.6 (Build 34)
```

Hi! Thank you so much for the great app! However I'm experiencing crash when modifying file name. The issue is not a very big deal to me (since I seldom change file names), but I can reproduce it every time in my current setting.

To reproduce: Enter the detail page of a password entry -> Tap Edit -> Change the "NAME" value -> Tap Save -> Screen turns black and app crashes

Thanks! I'm happy to provide more info if you have problem reproducing it.

**Comment by yishilin14** [comment]
Thank you. I just cheked this case. 

This issue exists in b48a7438ed4978b3fb74f26591cb50034633a70a (App Build 0.2.6(34)), and was later fixed by 7bfc3cc7d45a84bc2efcdbe5d88a394d38e7bdbc.

So the problem will be solved in our next App Store release. Stay tuned, I will close this issue once the next release is out.

**Comment by yishilin14** [comment]
Deleting an entry also crashes the app in 0.2.6(34), and will be fixed in our next public release.

**Comment by yishilin14** [comment]
Fixed in v0.2.7 in app store.
